extensions/todoist/src/components/MenubarTask.tsx

```diff
@@ -112,6 +114,16 @@ const MenuBarTask = ({ task, data, setData }: MenuBarTaskProps) => {
         ) : (
           <MenuBarExtra.Item title="Unfocus Task" onAction={() => unfocusTask()} icon={Icon.MinusCircle} />
         )}
+        {subTasks && subTasks?.length > 0 ? (
+          <MenuBarExtra.Submenu
+            title={`${subTasks.length} subtasks`}
```

Comment:
Could you make it so `subtask` is displayed instead of `subtasks` when there's only one sub-task?